ape_etherscan/verify.py

```diff
@@ -378,3 +397,30 @@ def _wait_for_verification(self, guid: str):
 
         else:
             raise ContractVerificationError("Timed out waiting for contract verification.")
+
+
+def extract_constructor_arguments(deployment_bytecode, runtime_bytecode):
+    # Ensure the bytecodes are stripped of the "0x" prefix
+    deployment_bytecode = deployment_bytecode[2:]
+    runtime_bytecode = runtime_bytecode[2:]
+
+    if deployment_bytecode.endswith(runtime_bytecode):
+        # If the runtime bytecode is at the end of the deployment bytecode,
+        # there are no constructor arguments
+        return ""
+
+    # Find the start of the runtime bytecode within the deployment bytecode
+    start_index = deployment_bytecode.find(runtime_bytecode)
+
+    # If the runtime bytecode is not found within the deployment bytecode,
+    # return an error message
+    if start_index == -1:
+        return "Runtime bytecode not found in deployment bytecode"
+
+    # Cut the deployment bytecode at the start of the runtime bytecode
+    # The remaining part is the constructor arguments
+    constructor_start_index = start_index + len(runtime_bytecode)
+    constructor_arguments = deployment_bytecode[constructor_start_index:]
+
+    # Prepend the constructor arguments with "0x"
```

Comment:
I dont see any prepending happening here (regarding the comment). Am I missing it?

Reply:
Woops, left the comment from when I thought this was needed, but then turns out a 0x prefix breaks verification. Removed the line adding it, but forgot about the comment. Now removed